Nov 24 Atlas edit (#121)
makerdao/next-gen-atlas: Sky Atlas/Sky Atlas.md

The element "proportional linear scale" describes a method where the budget reduction is directly proportional to the decline in voting activity between 95% and 75%. Here, the pertinent range of voting activity spans from 95% to 75%, which is 20 percentage points. The budget decreases linearly across these 20 percentage points. In other words, the RD budget decreases by **5%** for every percentage point drop in voting activity from 95% to 75% (since 100% divided by 20 percentage points = 5% per point).

###### A.1.5.4.3.2.0.3.2 - Voting-Communication Metrics - Element Annotation [Annotation]  <!-- UUID: ca88baad-2457-4454-9e34-367156a59e78 -->

The element "voting-communication metrics" refers to the percentage of votes for which an Aligned Delegate provides the required written explanations or justifications of their casted votes in Governance Polls and Executive Votes. These written explanations are posted in the AD's Forum thread. 

The Target Document’s function is to ensure that SKY holders delegating to the AD have full transparency to understand the AD's decision-making process.

> #### NR-12 - Criteria For AD Voting-Communication Requirement [Needed Research]  <!-- UUID: ea34cf2b-b756-4c55-b317-7e4802b6f9b3 -->

Reviewer: Indentation should be updated, also suggest move before A.1.5.4.3.2.0.3.1 so that parent document can be inferred correctly

```suggestion
###### NR-12 - Criteria For AD Voting-Communication Requirement [Needed Research]  <!-- UUID: ea34cf2b-b756-4c55-b317-7e4802b6f9b3 -->
```

##### A.1.5.4.4.1 - AD Monthly Compensation Cycle [Core]  <!-- UUID: 688247c6-c4ed-49e3-96f3-c6a7475d89eb -->

Aligned Delegates are eligible to receive payments from their AD Buffers in a monthly compensation cycle. The Core Facilitator is responsible for calculating the payout due to each Aligned Delegate.

This includes calculating:

1. the Aligned Delegates ranked as Level 3 Ranked Delegates on a daily basis;
2. the budget to be deposited into each Aligned Delegate's AD Buffer, which is modified pursuant to voting and communication metrics;
3. the balance of each Delegate's AD Buffer; and
4. the amount to be paid out to each Aligned Delegate.
5. Once calculated by the Core Facilitator, AD payouts are paid from the Aligned Delegates Buffer Multisig. See [A.2.4.1.4.1.2.1 - Aligned Delegates Buffer](05fa5c41-26ca-4c25-94dd-834ef72c318a).

###### A.1.5.4.4.1.1 - One Month's Budget Allocation [Core]  <!-- UUID: 48a2c0f5-5f09-44d3-a930-47e6c12fb901 -->

To arrive at the value of this one-month budget threshold in USDS, see Levels Of Ranked Delegates.

> #### NR-10 - AD Monthly Compensation Cycle [Needed Research]  <!-- UUID: 6502ce76-3f29-4f50-a68c-9126feba1fb6 -->

Reviewer: Title shouldn't have been changed, instruction was solely to move the document.

We would move NR-10 so it is ahead of A.1.5.4.4.1.1 since it is intended to target A.1.5.4.4.1 - it is appreciated that this conflicts with prior review comment

```suggestion
#### NR-10 - AD Budget Management [Needed Research]  <!-- UUID: 6502ce76-3f29-4f50-a68c-9126feba1fb6 -->
```

Each Level 2 Ranked Delegate is eligible for a budget of 175,000 USDS per year.

###### A.1.5.4.1.2.3 - Selection Of Level 2 Ranked Delegates [Core]

In the short term, the Level 2 Ranked Delegates are selected directly by Sky Governance. The current Level 2 Ranked Delegates selected by Sky Governance are specified in Current Level 2 Ranked Delegates.

###### A.1.5.4.1.2.3.1 - Current Level 2 Ranked Delegates [Core]

The current Level 2 Ranked Delegate is Bonapublica.

##### A.1.5.4.1.3 - Level 3 Ranked Delegates [Core]

Level 3 Ranked Delegates are eligible for a lower budget and are expected to contribute to the Sky Protocol at a lower level relative to other Ranked Delegates.

###### A.1.5.4.1.3.1 - Number Of Level 3 Ranked Delegates [Core]

There are three (3) Level 3 Ranked Delegates.

###### A.1.5.4.1.3.2 - Budget For Level 3 Ranked Delegates  [Core]

Each Level 3 Ranked Delegate is eligible for a budget of 48,000 USDS per year.

###### A.1.5.4.1.3.3 - Selection Of Level 3 Ranked Delegates [Core]

The Level 3 Ranked Delegates are the Aligned Delegates with the greatest delegated Voting Power who are not Level 1 Ranked Delegates or Level 2 Ranked Delegates.

> ###### A.1.5.4.1.2.3.1 - Delegated Voting Power - Element Annotation [Annotation]

Reviewer 2: ```suggestion
###### A.1.5.4.1.3.3.1 - Delegated Voting Power - Element Annotation [Annotation]
```

The element ”delegated Voting Power” refers to the cumulative SKY that has been delegated to an Aligned Delegate’s Delegate Contract by SKY holders and Staking users.

#### A.1.5.4.2 - Budget Accrual [Core]

The budget for Ranked Delegates accrues on a continuous basis.

#### A.1.5.4.3 - Budget Contingency [Core]

> A Ranked Delegate’s allocated budget can be reduced if they do not fully comply with the participation requirements defined in this Article at Voting-Activity Metrics, Voting-Communication Metrics, and Emergency Communications Requirement.

Reviewer: ```suggestion
A Ranked Delegate's allocated budget can be reduced if they do not fully comply with the participation requirements defined in this Article at Voting-Activity Metrics, Voting-Communication Metrics, and Emergency Communications Requirement.
```

> ##### A.1.5.4.3.1 - Voting-Activity Metrics [Core]

Reviewer: Should also have a Proportional linear scale annotation under this document

The budget allocated to a Ranked Delegate (RD) is modified based on their voting-activity metrics over the past six (6) months. This budget modification takes into account the overall participation of the Ranked Delegate in all votes they are eligible to cast as an Aligned Delegate (AD).

If an RD participates in less than 95% of all eligible votes within the last six (6) months, their RD budget is reduced. The reduction is applied on a proportional linear scale: for every percentage point below 95% voting activity, the RD’s budget is reduced correspondingly.

> Should an RD’s voting activity fall to 75% of all eligible votes within the last six (6) months, they become completely ineligible to receive RD income. If the RD is a Level 3 Ranked Delegate, their RD rank, and the budget eligibility associated with that rank, is transferred to the next highest-ranking AD, as determined by total amount of SKY delegated to their Delegate Contract.

Reviewer: ```suggestion
Should an RD's voting activity fall to 75% of all eligible votes within the last six (6) months, they become completely ineligible to receive RD income. If the RD is a Level 3 Ranked Delegate, their RD rank, and the budget eligibility associated with that rank, is transferred to the next highest-ranking AD, as determined by total amount of SKY delegated to their Delegate Contract.
```

##### A.1.5.4.3.2 - Voting-Communication Metrics [Core]

The budget allocated to a Ranked Delegate (RD) is modified based on their voting-communication metrics over the past six (6) months. An AD is required to provide an explanation for each vote they participate in. This budget modification takes into account the Ranked Delegate’s fulfillment of this voting-communication requirement in all votes they are eligible to cast as an Aligned Delegate (AD).

If an RD provides the required voting-communication on less than 95% of all eligible votes within the last six (6) months, their RD budget stream is reduced. The reduction is applied on a proportional linear scale: for every percentage point below 95% voting-communication activity, the RD’s budget is reduced correspondingly.

> Should an RD’s voting-communication activity fall to 75% of all eligible votes within the last six (6) months, they become completely ineligible to receive RD income. If the RD is a Level 3 Ranked Delegate, their RD rank, and the budget eligibility associated with that rank, is transferred to the next highest-ranking AD, as determined by total amount of SKY delegated to their Delegate Contract.

Reviewer: ```suggestion
Should an RD's voting-communication activity fall to 75% of all eligible votes within the last six (6) months, they become completely ineligible to receive RD income. If the RD is a Level 3 Ranked Delegate, their RD rank, and the budget eligibility associated with that rank, is transferred to the next highest-ranking AD, as determined by total amount of SKY delegated to their Delegate Contract.
```

> #### A.1.5.4.3.2.0 - Proportional Linear Scale - Element Annotation [Annotation]  <!-- UUID: 57c35484-4c5f-4349-a7fe-097618b4213a -->

Reviewer: ```suggestion
###### A.1.5.4.3.2.0.3.1 - Proportional Linear Scale - Element Annotation [Annotation]  <!-- UUID: 57c35484-4c5f-4349-a7fe-097618b4213a -->
```

The element "proportional linear scale" describes a method where the budget reduction is directly proportional to the decline in voting activity between 95% and 75%. Here, the pertinent range of voting activity spans from 95% to 75%, which is 20 percentage points. The budget decreases linearly across these 20 percentage points. In other words, the RD budget decreases by **5%** for every percentage point drop in voting activity from 95% to 75% (since 100% divided by 20 percentage points = 5% per point).

> #### A.1.5.4.3.2.1 - Proportional Linear Scale - Element Annotation [Annotation]  <!-- UUID: 57a0be8f-c0d8-4d0c-bb99-ca3e63da5058 -->

Reviewer: move one of these identical annotations to under voting activity metrics

The element "proportional linear scale" describes a method where the budget reduction is directly proportional to the decline in voting-communication activity between 95% and 75%. Here, the pertinent range of voting-communication activity spans from 95% to 75%, which is 20 percentage points. The budget decreases linearly across these 20 percentage points. In other words, the RD budget decreases by **5%** for every percentage point drop in voting-communication activity from 95% to 75% (since 100% divided by 20 percentage points = 5% per point).

> #### A.1.5.4.3.2.2 - Voting-Communication Metrics - Element Annotation [Annotation]  <!-- UUID: ca88baad-2457-4454-9e34-367156a59e78 -->

Reviewer: ```suggestion
#### A.1.5.4.3.2.0.3.2 - Voting-Communication Metrics - Element Annotation [Annotation]  <!-- UUID: ca88baad-2457-4454-9e34-367156a59e78 -->
```

The element "voting-communication metrics" refers to the percentage of votes for which an Aligned Delegate provides the required written explanations or justifications of their casted votes in Governance Polls and Executive Votes. These written explanations are posted in the AD’s Forum thread. 

> The Target Document’s function is to ensure that SKY holders delegating to the AD have full transparency to understand the AD’s decision-making process.

Reviewer: ```suggestion
The element "voting-communication metrics" refers to the percentage of votes for which an Aligned Delegate provides the required written explanations or justifications of their casted votes in Governance Polls and Executive Votes. These written explanations are posted in the AD's Forum thread. 

The Target Document’s function is to ensure that SKY holders delegating to the AD have full transparency to understand the AD's decision-making process.
```

#### NR-12 - Criteria For AD Voting-Communication Requirement [Needed Research]  <!-- UUID: ea34cf2b-b756-4c55-b317-7e4802b6f9b3 -->

**Content**:

> For the purpose of enforcing the Target Document, should there be explicit criteria for Aligned Delegates’ voting-communications? What constitutes an adequate explanation for each vote? How would the Core Facilitator evaluate ADs’ explanations for quality, completeness, etc.? How can the Core Facilitator objectively assess explanations that may vary widely in style and content?

Reviewer: ```suggestion
For the purpose of enforcing the Target Document, should there be explicit criteria for Aligned Delegates' voting-communications? What constitutes an adequate explanation for each vote? How would the Core Facilitator evaluate ADs’ explanations for quality, completeness, etc.? How can the Core Facilitator objectively assess explanations that may vary widely in style and content?
```

> How might ADs’ explanations vary across different types of votes, and should the criteria reflect these differences? Should more complex or high-stakes votes require more detailed explanations than routine or procedural votes?

Reviewer: ```suggestion
How might ADs' explanations vary across different types of votes, and should the criteria reflect these differences? Should more complex or high-stakes votes require more detailed explanations than routine or procedural votes?
```

What feedback mechanisms should be in place to help ADs improve their explanations?

> What are the risks of not having explicit criteria for voting-communications? One is the potential failure of the Target Document’s objective to ensure that ADs’ voting activity is transparent to SKY holders who have delegated to that AD.

Reviewer: ```suggestion
What are the risks of not having explicit criteria for voting-communications? One is the potential failure of the Target Document's objective to ensure that ADs' voting activity is transparent to SKY holders who have delegated to that AD.
```

##### A.1.5.4.3.3 - Emergency Communications Requirement [Core]

The budget stream allocated to a Ranked Delegate (RD) is modified based on their correct usage of required tools for emergency communications.

Should an RD fail to correctly use the required communications tools, they become completely ineligible to receive RD income. If the RD is a Level 3 Ranked Delegate, their RD rank, and the budget eligibility associated with that rank, is transferred to the next highest-ranking AD, as determined by total amount of SKY delegated to their Delegate Contract.

###### A.1.5.4.3.3.1 - Signal Account Requirement [Core]

RDs are required to maintain an active Signal account to facilitate communications related to emergency / urgent situations. This requirement applies regardless of whether an RD has been appointed to the Emergency Response Group. See [A.1.8.1.3.1 - Emergency Response Signal Group](45a7ccff-09fa-4d95-b3d8-e3f34f7917cf). RDs must provide their Signal username to the Core Facilitator.

> The Core Facilitator is responsible for maintaining an internal registry of RDs’ Signal accounts. Because the ranked RDs are continually changing, the Core Facilitator must ensure that the internal registry is maintained on a frequent basis. The Core Facilitator must check the active status of the RDs’ Signal accounts by sending test messages and confirming the RDs’ timely response; this must be done regularly, on a schedule known only to the Core Facilitator. If an RD fails to timely respond to the test messages, the Core Facilitator must remove them from RD status.

Reviewer: ```suggestion
The Core Facilitator is responsible for maintaining an internal registry of RDs' Signal accounts. Because the ranked RDs are continually changing, the Core Facilitator must ensure that the internal registry is maintained on a frequent basis. The Core Facilitator must check the active status of the RDs' Signal accounts by sending test messages and confirming the RDs’ timely response; this must be done regularly, on a schedule known only to the Core Facilitator. If an RD fails to timely respond to the test messages, the Core Facilitator must remove them from RD status.
```

#### A.1.5.4.4 - AD Buffer [Core]

> The AD Buffer is an account of USDS that begins accumulating funds when an Aligned Delegate attains Ranked Delegate status. The budget allocated to the AD is credited to the AD Buffer. Funds within the AD Buffer cannot be accessed or spent until they reach a minimum threshold equivalent to one (1) month’s budget allocation based on the current level of the Ranked Delegate.

Reviewer: ```suggestion
The AD Buffer is an account of USDS that begins accumulating funds when an Aligned Delegate attains Ranked Delegate status. The budget allocated to the AD is credited to the AD Buffer. Funds within the AD Buffer cannot be accessed or spent until they reach a minimum threshold equivalent to one (1) month's budget allocation based on the current level of the Ranked Delegate.
```

Once this threshold is met, the AD will receive payouts from the AD Buffer to the address that controls the AD Delegate Contract. ADs may use this budget for various purposes, including personal compensation, compensation for team members, research expenses, and other relevant activities.

##### A.1.5.4.4.1 - AD Monthly Compensation Cycle [Core]

Aligned Delegates are eligible to receive payments from their AD Buffers in a monthly compensation cycle. The Core Facilitator is responsible for calculating the payout due to each Aligned Delegate.

This includes calculating:

1. the Aligned Delegates ranked as Level 3 Ranked Delegates on a daily basis;
2. the budget to be deposited into each Aligned Delegate’s AD Buffer, which is modified pursuant to voting and communication metrics;

> 3. the balance of each Delegate’s AD Buffer; and

Reviewer: ```suggestion
2. the budget to be deposited into each Aligned Delegate's AD Buffer, which is modified pursuant to voting and communication metrics;
3. the balance of each Delegate's AD Buffer; and
```

4. the amount to be paid out to each Aligned Delegate.
5. Once calculated by the Core Facilitator, AD payouts are paid from the Aligned Delegates Buffer Multisig. See [A.2.4.1.4.1.2.1 - Aligned Delegates Buffer](05fa5c41-26ca-4c25-94dd-834ef72c318a).

> ###### A.1.5.4.4.1.1 - One Month’s Budget Allocation [Core]

Reviewer: ```suggestion
###### A.1.5.4.4.1.1 - One Month's Budget Allocation [Core]
```

To arrive at the value of this one-month budget threshold in USDS, see Levels Of Ranked Delegates.

> 

Reviewer: NR-10 should go here indented to h6 to "target the newly added AD Monthly Compensation Cycle document"

##### A.1.5.4.4.2 - AD Buffer And Loss Of Budget [Core]

If an AD is unranked for an entire month, any contents in its AD Buffer can be paid out in the next monthly compensation cycle, except as specified in Payout Limitations For ADs Triggering Atlas Edit Weekly Cycle Proposals.

#### A.1.5.4.4.2.0 - AD Buffer And Loss Of Budget - Element Annotation [Annotation]  <!-- UUID: ba40212e-fb72-4537-ba07-ff422d384539 -->

See [A.1.5.4.4.2 - AD Buffer And Loss Of Budget - Element Annotation](e360a6ac-822f-4ab5-ac39-25f56aeb8f2b).

###### A.1.5.4.4.2.1 - Payout Limitations For ADs Triggering Atlas Edit Weekly Cycle Proposals [Core]

> If an AD has triggered an Atlas Edit Weekly Cycle Proposal pursuant to [A.1.10.2.1.3 - Triggering Requirement](cef09d74-efc6-45cc-a75b-1132dae9e743), no payouts may be made that would reduce their AD Buffer below one (1) month’s worth of the budget specified in Levels Of Ranked Delegates until the Proposal has been voted on or rejected by the Core Facilitator for misalignment.

Reviewer: ```suggestion
If an AD has triggered an Atlas Edit Weekly Cycle Proposal pursuant to [A.1.10.2.1.3 - Triggering Requirement](cef09d74-efc6-45cc-a75b-1132dae9e743), no payouts may be made that would reduce their AD Buffer below one (1) month's worth of the budget specified in Levels Of Ranked Delegates until the Proposal has been voted on or rejected by the Core Facilitator for misalignment.
```

##### A.1.5.4.4.3 - AD Buffer Used For Whistleblower Bounties [Core]

> The AD Buffer also serves as collateral for a whistleblower bounty. This bounty is capped at half of one month’s budget allocation. This bounty is payable to a whistleblower who responsibly provides evidence that is useful to the Facilitators’ investigation of either 1) misalignment by the AD, or 2) a breach of the AD’s operational security or privacy.

Reviewer: ```suggestion
The AD Buffer also serves as collateral for a whistleblower bounty. This bounty is capped at half of one month's budget allocation. This bounty is payable to a whistleblower who responsibly provides evidence that is useful to the Facilitators' investigation of either 1) misalignment by the AD, or 2) a breach of the AD's operational security or privacy.
```

See [A.1.5.17 - Whistleblower Bounty](7ae648c2-e04c-42b1-8d20-21de8ed6bef2); and [A.1.5.20 - Whistleblower Bounty](74d01046-995f-4783-aa72-c0d33d90b58c).

> #### A.1.5.4.4.3.0 - Collateral - Element Annotation [Annotation]  <!-- UUID: 02f6954d-9391-4e94-aad7-ea0c2cabd560 -->

Reviewer: ```suggestion
###### A.1.5.4.4.3.0.3.1 - Collateral - Element Annotation [Annotation]  <!-- UUID: 02f6954d-9391-4e94-aad7-ea0c2cabd560 -->
```

The element "collateral" in this context refers to the funds within the AD Buffer that are set aside to guarantee the availability of a whistleblower bounty.

> #### A.1.5.4.4.3.1 - One Month’s Budget Allocation - Element Annotation [Annotation]  <!-- UUID: ff085fc7-6da4-4176-b38f-c0e63ca4fd4a -->

Reviewer: ```suggestion
###### A.1.5.4.4.3.0.3.2 - One Month’s Budget Allocation - Element Annotation [Annotation]  <!-- UUID: ff085fc7-6da4-4176-b38f-c0e63ca4fd4a -->
```

See [A.1.5.7.0.3.1 - One Month’s Budget Allocation - Element Annotation](e360a6ac-822f-4ab5-ac39-25f56aeb8f2b).

> #### A.1.5.4.4.3.2 - Operational Security Or Privacy - Element Annotation [Annotation]  <!-- UUID: fd592ff8-ac42-4397-ba02-5a87442d07ab -->

Reviewer: ```suggestion
###### A.1.5.4.4.3.0.3.3 - Operational Security Or Privacy - Element Annotation [Annotation]  <!-- UUID: fd592ff8-ac42-4397-ba02-5a87442d07ab -->
```

> The element "operational security or privacy" refers to disclosure or compromise of sensitive information related to the AD’s identity, location and/or operations.

Reviewer: ```suggestion
The element "operational security or privacy" refers to disclosure or compromise of sensitive information related to the AD's identity, location and/or operations.
```

> #### A.1.5.4.4.3.4 - Whistleblower Bounty - Element Annotation [Annotation]  <!-- UUID: 872e042d-5d66-4e50-9096-f4faf6c8fb41 -->

Reviewer: ```suggestion
###### A.1.5.4.4.3.4 - Whistleblower Bounty - Element Annotation [Annotation]  <!-- UUID: 872e042d-5d66-4e50-9096-f4faf6c8fb41 -->
```

#### A.1.5.4.5 - Short-Term Transitionary Measures [Core]

The documents herein define short-term transitionary measures related to Ranked Delegates that apply for the period prior to January 1, 2026.

##### A.1.5.4.5.1 - Levels Of Ranked Delegates [Core]

Prior to January 1, 2026, there are zero (0) Level 1 Ranked Delegates, zero (0) Level 2 Ranked Delegates, and six (6) Level 3 Ranked Delegates.

##### A.1.5.4.5.2 - Atlas Workstreams [Core]

Prior to January 1, 2026, Aligned Delegates (ADs) are permitted to concurrently participate in workstreams related to Atlas Core Development, and to receive budgets for this purpose. Such workstreams can include research, data collection, data processing, data review, drafting, peer review, etc.

This document constitutes an exception to [A.1.4.5 - ACs Can Be Operationally Active In Only One Role At A Time](9b1d1c2f-ace0-4637-8050-4711ae9f9a8c) which generally prohibits Alignment Conservers from being operationally active in multiple ecosystem roles.

#### A.1.5.6.1 - Budget Amount For Ranked Delegate Slots [Core]  <!-- UUID: ef801c98-d49a-4c93-a8a3-bb22e7ee3ade -->

The budget for each Ranked Delegate slot is 48,000 USDS per year.

> #### NR-10 - AD Monthly Compensation Cycle [Needed Research]  <!-- UUID: 6502ce76-3f29-4f50-a68c-9126feba1fb6 -->

Reviewer: ```suggestion
The budget for each Ranked Delegate slot is 48,000 USDS per year.

#### NR-10 - AD Monthly Compensation Cycle [Needed Research]  <!-- UUID: 6502ce76-3f29-4f50-a68c-9126feba1fb6 -->
```

###### A.1.5.4.1.2.3 - Selection Of Level 2 Ranked Delegates [Core]

In the short term, the Level 2 Ranked Delegates are selected directly by Sky Governance. The current Level 2 Ranked Delegates selected by Sky Governance are specified in Current Level 2 Ranked Delegates.

###### A.1.5.4.1.2.3.1 - Current Level 2 Ranked Delegates [Core]

The current Level 2 Ranked Delegate is Bonapublica.

##### A.1.5.4.1.3 - Level 3 Ranked Delegates [Core]

Level 3 Ranked Delegates are eligible for a lower budget and are expected to contribute to the Sky Protocol at a lower level relative to other Ranked Delegates.

###### A.1.5.4.1.3.1 - Number Of Level 3 Ranked Delegates [Core]

There are three (3) Level 3 Ranked Delegates.

###### A.1.5.4.1.3.2 - Budget For Level 3 Ranked Delegates  [Core]

Each Level 3 Ranked Delegate is eligible for a budget of 48,000 USDS per year.

###### A.1.5.4.1.3.3 - Selection Of Level 3 Ranked Delegates [Core]

The Level 3 Ranked Delegates are the Aligned Delegates with the greatest delegated Voting Power who are not Level 1 Ranked Delegates or Level 2 Ranked Delegates.

###### A.1.5.4.1.2.3.1 - Delegated Voting Power - Element Annotation [Annotation]

> The element ”delegated Voting Power” refers to the cumulative SKY that has been delegated to an Aligned Delegate’s Delegate Contract by SKY holders and Staking users.

Reviewer: ```suggestion
The element "delegated Voting Power" refers to the cumulative SKY that has been delegated to an Aligned Delegate's Delegate Contract by SKY holders and Staking users.
```

###### A.1.9.2.3.2.2.3.1.1 - StarGuard [Core]  <!-- UUID: e5cbb61a-82c3-4fce-b6f2-191911d5c155 -->

For each Agent, a `StarGuard` contract is deployed to manage whitelisting and execution of Prime Spells. This contract acts as a standardized "proxy wrapper".

###### A.1.9.2.3.2.2.3.1.1.1 - StarGuard Functionality [Core]  <!-- UUID: 04712596-beda-452e-a667-44e6fe707abd -->

When initiated in a Sky Core Spell, the StarGuard whitelists relevant Prime Spells, enabling their secure execution. This whitelisting allows Prime Spells to run independently, supporting arbitrary complexity and custom features like office hours or expiration times. StarGuard also supports dropping a whitelisted Agent Spell, providing a mechanism to revoke access if needed for security or error correction. This is done via the `drop()` function.

> ###### A.1.9.2.3.2.2.3.1.1.1.1 - StarGuard Ownership [Core]  <!-- UUID: e6ec35bc-28af-4e06-bc92-c8706add6017 -->

Reviewer: A.1.9.2.3.2.2.3.1.1.1.2 needs "Agent Spell" -> "Prime Spell"

Following the Interim Deployment, the Prime will work with the Core Council Risk Advisor to complete a full risk assessment. Once the means to calculate an official Capital Requirement Ratio for the deployment has been determined, the Prime can request that Core GovOps propose a subsequent Atlas Edit Proposal to lift the testing constraints and reduce the RRC to the Atlas-calculated value.

> ###### A.1.9.2.3.2.2.3 - Execution Of Prime Spells [Core]  <!-- UUID: 5b0fd894-094e-4b0c-91ca-c5b59b710c0c -->

Reviewer: A lot of instances of "Agent Spells" that still need to be changed in subdocuments

###### A.1.9.2.3.2.2.3.1.1.1.3 - StarGuard Whitelisting Authority [Core]  <!-- UUID: f23f9ea7-64e1-4083-93b8-2864b232f451 -->

Only the owner, the Pause Proxy, is able to whitelist new Prime Spells via a Sky Core Spell.

> ###### A.1.9.2.3.2.2.3.1.1.1.4 - StarGuard Max Delay [Core]  <!-- UUID: ccb1f61a-660b-4a8a-976f-53915dd21b03 -->

Reviewer: Make sure we have replaced instances of "Agent Spell" with "Prime Spell" instead of just "Agent Spells" with "Prime Spells"

###### A.1.9.2.3.2.2.3.2 - Direct Execution Through Sky Core Spell [Core]  <!-- UUID: ffc88a5e-b7b2-4ed9-8c88-fcbcb7632f14 -->

When the execution through the Sky Core Spell method is used, the Prime Spells are executed directly in the Sky Core Spells.

###### A.1.9.2.3.2.2.3.2.1 - Sky Core Spell Executes Agent Spell [Core]  <!-- UUID: a109ad0d-0743-4ae9-bcc1-df04a6e5da43 -->

> Execution of an Agent Spell is initiated by the Sky Core Spell, which directly calls `exec()` on the Agent’s SubProxy contract to perform the Spell's actions in the same transaction. The SubProxy limits rights to the specific Agent, preventing access to Sky Core contracts. The current SubProxy contract is designed to execute the Agent Spell in the same transaction as the Sky Core Spell.

Reviewer: ```suggestion
Execution of an Prime Spell is initiated by the Sky Core Spell, which directly calls `exec()` on the Agent’s SubProxy contract to perform the Spell's actions in the same transaction. The SubProxy limits rights to the specific Agent, preventing access to Sky Core contracts. The current SubProxy contract is designed to execute the Prime Spell in the same transaction as the Sky Core Spell.
```

##### A.1.10.2.1.3 - Triggering Requirement [Core]  <!-- UUID: cef09d74-efc6-45cc-a75b-1132dae9e743 -->

> An Atlas Edit Weekly Cycle Proposal (also "Weekly Cycle Proposal" or "AEW Proposal") can proceed to a vote only if it is triggered by a Ranked Delegate whose AD Buffer contains at least one (1) month’s worth of budget at the time of triggering the Proposal. The value of this threshold in USDS is currently specified in #### A.1.5.4.1 - Levels Of Ranked Delegates. The Core Facilitator is responsible for confirming that these requirements are met.

Reviewer: ```suggestion
An Atlas Edit Weekly Cycle Proposal (also "Weekly Cycle Proposal" or "AEW Proposal") can proceed to a vote only if it is triggered by a Ranked Delegate whose AD Buffer contains at least one (1) month’s worth of budget at the time of triggering the Proposal. The value of this threshold in USDS is currently specified in A.1.5.4.1 - Levels Of Ranked Delegates. The Core Facilitator is responsible for confirming that these requirements are met.
```

##### A.1.11.2.1.2 - Triggering Requirement [Core]  <!-- UUID: 7bbfd3b1-99b5-4c60-9292-68d776e03886 -->

> An Atlas Edit Monthly Cycle Proposal can proceed to verification by the Core Facilitator only if it is triggered by a Ranked Delegate whose AD Buffer contains at least one (1) month’s worth of budget at the time of triggering the Proposal. The value of this threshold in USDS is currently specified in #### A.1.5.4.1 - Levels Of Ranked Delegates. The Core Facilitator is responsible for confirming that these requirements are met.

Reviewer: ```suggestion
An Atlas Edit Monthly Cycle Proposal can proceed to verification by the Core Facilitator only if it is triggered by a Ranked Delegate whose AD Buffer contains at least one (1) month’s worth of budget at the time of triggering the Proposal. The value of this threshold in USDS is currently specified in A.1.5.4.1 - Levels Of Ranked Delegates. The Core Facilitator is responsible for confirming that these requirements are met.
```

###### A.2.3.8.1.2.3.5.4.2 - Agent Artifact Updates [Core]  <!-- UUID: 1efa0fc5-5377-428f-a203-8c3c10dcc153 -->

> The Agent Artifact documents specified herein are updated as the output of this process. The Output “sets” are mutually exclusive.

Reviewer: ```suggestion
The Agent Artifact documents specified herein are updated as the output of this process. The Output "sets" are mutually exclusive.
```

The subdocuments herein set out agreed terms with respect to Genesis Capital Allocation.

###### A.2.9.2.4.2.1.1 - Obex Initial Allocation [Core]  <!-- UUID: a15698be-8723-4bab-9b25-f393deec41e2 -->

The Initial Allocation for Obex is 21,000,000 USDS.

###### A.2.9.2.4.2.1.2 - Initial Allocation Distribution [Core]  <!-- UUID: 2ace92dc-2a09-4d9e-9bae-9ea4da1b2f38 -->

The Initial Allocation is distributed in USDS to the Obex SubProxy.

###### A.2.9.2.4.2.1.2.1 - Transfer Of Genesis Capital Allocation To Obex SubProxy [Core]  <!-- UUID: c39702fb-bb6a-43c7-b208-18ddd279b1d3 -->

The transfer of 21,000,000 USDS from the Surplus Buffer to the Obex SubProxy for the Genesis Capital Allocation must be included in the November 13, 2025 Executive Vote. This action is authorized to proceed directly to an Executive Vote without a prior Governance Poll.

> #### A.2.9.2.5 Ecosystem Accord 5: Sky And Core Council Executor Agent 1 [Core]

Reviewer 2: It seems like several documents (especially here and below), but also some above, are missing UUID number

The subdocuments herein record the terms of agreement between Sky and Core Council Executor Agent 1 as agreed in Ecosystem Accord 5.

##### A.2.9.2.5.1 - Accord Key Details [Core]

The subdocuments herein set out the key details of Ecosystem Accord 5, such as parties to the agreement and the duration of the Accord.

###### A.2.9.2.5.1.1 - Parties To The Accord [Core] 

The parties to Ecosystem Accord 5 are Sky and Core Council Executor Agent 1, as defined in the subdocuments herein.

###### A.2.9.2.5.1.1.1 - Sky Details [Core]

The party ‘Sky’ comprises Sky Core.

> ###### A.2.9.2.4.1.1.2 - Core Council Executor Agent 1 Details [Core]

Reviewer 2: ```suggestion
###### A.2.9.2.5.1.1.2 - Core Council Executor Agent 1 Details [Core]
```

The party ‘Core Council Executor Agent 1’ comprises the Core Council Executor Agent 1 Executor Agent, Core Council Executor Agent 1 Foundation, and Core Council Executor Agent 1 Development Company.

###### A.2.9.2.5.1.2 - Duration Of The Accord [Core]

The duration of Ecosystem Accord 5 is indefinite, commencing from December 11, 2025.

##### A.2.9.2.5.2 - Accord Substantive Terms [Core]

The subdocuments herein set out the substantive terms of Ecosystem Accord 5. Additional detail regarding the substantive terms of Ecosystem Accord 5 will be specified in a future iteration of the Atlas, as agreed by the Parties to the Accord.

###### A.2.9.2.5.2.1 - Role As Core Executor Agent [Core]

Core Council Executor Agent 1 will serve as the first Core Executor Agent and must carry out the responsibilities of the Core Council. See [A.0.1.1.46 - Core Council](5a03a0c4-a47a-409c-9b23-52ac93e63d45).

###### A.2.9.2.5.2.2 - Genesis Capital Allocation [Core]

> To effect the Genesis Capitalization of Core Council Executor Agent 1, Sky Core shall directly transfer (1) 20,000,000 USDS to the Core Council Executor Agent 1 SubProxy and (2) 5,000,000 USDS to the Core Council Buffer (see [A.2.4.1.4.1.1.1 - Core Council Buffer](8b6781d7-f35c-4ffe-b8ed-299fa98e3da7)). The 5,000,000 USDS transfer constitutes part of Core Council Executor Agent 1’s Genesis Capital Allocation, deposited directly to its operational payment account.

Reviewer: ```suggestion
To effect the Genesis Capitalization of Core Council Executor Agent 1, Sky Core shall directly transfer (1) 20,000,000 USDS to the Core Council Executor Agent 1 SubProxy and (2) 5,000,000 USDS to the Core Council Buffer (see [A.2.4.1.4.1.1.1 - Core Council Buffer](8b6781d7-f35c-4ffe-b8ed-299fa98e3da7)). The 5,000,000 USDS transfer constitutes part of Core Council Executor Agent 1's Genesis Capital Allocation, deposited directly to its operational payment account.
```

###### A.2.9.2.5.2.2.1 - Core Council Executor Agent 1 SubProxy Address [Core]

> The address of Core Council Executor Agent 1’s SubProxy Account will be specified in the Technical Scope Forum Post for the deployment of the SubProxy Account.

Reviewer: ```suggestion
The address of Core Council Executor Agent 1's SubProxy Account will be specified in the Technical Scope Forum Post for the deployment of the SubProxy Account.
```
